guppylang/std/builtins.py

```diff
@@ -71,13 +71,20 @@
 L = guppy.type_var("L", copyable=False, droppable=False)
 
 
-def comptime(*args: Any) -> Any:
-    """Function to tag compile-time evaluated Python expressions in a Guppy context.
+class _Comptime:
+    """Dummy class to support `@comptime` annotations and `comptime(...)` expressions."""
+
+    def __call__(self, *args: Any) -> Any:
+        return tuple(args)
+
+    def __rmatmul__(self, other: Any) -> Any:
+        return other
```

Comment:
This is to make the Python interpreter happy with `@comptime` at runtime

Reply:
this would be better as a comment in the code